egs/babel_universal_am/s5/local/prepare_universal_lexicon.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
```

Comment:
is this ever used? we are running phonemic systems...

Reply:
it's universal, not unicode. I was uncreative with the name.